packages/plugin-vue/src/index.ts

```diff
@@ -201,6 +203,12 @@ export default function vuePlugin(rawOptions: Options = {}): Plugin {
       }
 
       const { filename, query } = parseVueRequest(id)
+
+      // external scoped style
+      if (!query.vue && query.scoped) {
```

Comment:
I'm afraid this design may be too intrusive.
It hijacks all requests with a `scoped` query parameter, no matter if it's requested from a Vue SFC, or if it's a CSS request.
Considering there are other flavors of scoped CSS in other frameworks, we should restrict this feature for Vue only.